docs/stoploss.md

```diff
@@ -22,14 +22,18 @@ These modes can be configured with these values:
     'stoploss_on_exchange_limit_ratio': 0.99
 ```
 
-### stoploss_on_exchange
+### stoploss_on_exchange and stoploss_on_exchange_limit_ratio
 Enable or Disable stop loss on exchange.
 If the stoploss is *on exchange* it means a stoploss limit order is placed on the exchange immediately after buy order happens successfully. This will protect you against sudden crashes in market as the order will be in the queue immediately and if market goes down then the order has more chance of being fulfilled.
 
 If `stoploss_on_exchange` uses limit orders, the exchange needs 2 prices, the stoploss_price and the Limit price.
 `stoploss` defines the stop-price - and limit should be slightly below this.  
 If an exchange supports both limit and market stoploss orders, then the value of `stoploss` will be used to determine the stoploss type.
 
+If `stoploss_on_exchange` fails to fill we can use `stoploss_on_exchange_limit_ratio` that defaults to 0.99 / 1% to perform an [emergency sell](#emergencysell).
```

Comment:
`stoploss_on_exchange_limit_ratio` has nothing to do with emergency sell... 

A limit stoploss order requires 2 prices - a stop-price (when the order will be placed) - and a limit price (up to which point the order can be filled).
Using the below numbers - your stop-price would be 95$ (so the order is placed once price crosses below 95$) and the limit price is 94.05$ (1% below 95%) - which means that the limit sell order will be placed at  94.05$. 
Most likely, the exchange will fill it right away (assuming the price sits between 95$ and 94.05$), unless price dips just for a microsecond - and goes above 95$ again, where the average price will be > stoploss (but that's not a problem, really).

This will protect you from VERY fast drops - as it'll guarantee you a exit price - but obviously you can remain stuck in a trade if price drops "past" your stoploss before it can be sold (although i don't think that's possible on binance - as binance would refuse such a limit order with "would trigger immediately" - which is when emergency sell comes into play - which is using the current market price to price the order (just like a regular sell order would) - but defaults to a market order ...


Kraken does not have any of this - but uses stoploss-market orders - which means once the order is placed (when price crosses below 95$) - it'll place a market sell order - which obviously fill at "whatever" price it finds volume in the orderbook.
Therefore, `stoploss_on_exchange_limit_ratio` will be ignored for stoploss-market order types